<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add</class>
 <widget class="QWidget" name="Add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>229</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="addressEdit">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>120</y>
     <width>128</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>15</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>58</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>8</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Score</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>104</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Map</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="scoreSpinBox">
   <property name="geometry">
    <rect>
     <x>154</x>
     <y>25</y>
     <width>63</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100000000</number>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelPushButton">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>199</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okPushButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>199</y>
     <width>76</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>8</y>
     <width>56</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Application</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="mapSpinBox">
   <property name="geometry">
    <rect>
     <x>154</x>
     <y>121</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="levelSpinBox">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>75</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>156</x>
     <y>59</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Level</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>75</y>
     <width>127</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>6</number>
   </property>
  </widget>
  <widget class="QComboBox" name="applicationComboBox">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>25</y>
     <width>126</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>RedSquare</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SpaceBubble</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TowerDefense</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>104</y>
     <width>64</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Address</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>153</x>
     <y>150</y>
     <width>64</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Version</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="versionEdit">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>166</y>
     <width>59</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>4</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>148</y>
     <width>64</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>165</y>
     <width>128</width>
     <height>22</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>dd-MM-yyyy HH:mm:ss</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>applicationComboBox</tabstop>
  <tabstop>scoreSpinBox</tabstop>
  <tabstop>nameEdit</tabstop>
  <tabstop>levelSpinBox</tabstop>
  <tabstop>addressEdit</tabstop>
  <tabstop>mapSpinBox</tabstop>
  <tabstop>dateTimeEdit</tabstop>
  <tabstop>versionEdit</tabstop>
  <tabstop>okPushButton</tabstop>
  <tabstop>cancelPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
